transform/models/marts/geo_reference/geo_reference__places_joined.sql

```diff
@@ -0,0 +1,44 @@
+with footprints as (
+    select
+        "release",
+        "capture_dates_range",
+        "geometry"
+    from {{ source('building_footprints', 'california_building_footprints') }}
+),
+
+places_source as (
+    select * from {{ source('tiger_2022', 'places') }}
+),
+
+places_renamed as (
+    select
+        "PLACEFP" as "place_fp",
+        "PLACENS" as "place_ns",
+        "GEOID" as "geoid",
+        "NAME" as "name",
+        "CLASSFP" as "class_fips_code",
+        "geometry"
+    from places_source
+),
+
+places as (
+    select
+        "place_fp",
+        "place_ns",
+        "geoid",
+        "name",
+        "class_fips_code",
+        {{ map_class_fips("class_fips_code") }} as "class_fips",
+        "geometry"
+    from places_renamed
+),
```

Comment:
Purely stylistic, but here and in the blocks model, I think these two CTEs could be combined. I don't think separating them increases legibility

Reply:
Agree! Fixed